### PR TITLE
budgetProjectV1

### PR DESCRIPTION
**Pull request by ZilvinasVysniauskas**
Add record functionality. Display record functionality. Delete record functionality. Edit record functionality. Display balance.

### CODE REVIEW

File: src/main/java/budgetApplication/BudgetApplication.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package budgetApplication;
+
+import budgetApplication.menu.userInput.EnterOperationChoice;
+import budgetApplication.menu.MainOperationStarter;
+
+public class BudgetApplication {
+
+
```
Comment (andzejc) [inline src/main/java/budgetApplication/BudgetApplication.java:8]: delete multiple empty lines

File: src/main/java/budgetApplication/menu/MainOperationStarter.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                break;
+            case "saveData":
+                new Files().saveData(budget.getIncomeRecords(), budget.getExpenseRecords());
+                System.out.println("records saved successfully");
+                break;
+            case "loadData":
+                new Files().loadData(budget);
+                System.out.println("records loaded successfully");
+                break;
+        }
```
Comment (andzejc) [inline src/main/java/budgetApplication/menu/MainOperationStarter.java:43]: create enums for cases

File: src/main/java/budgetApplication/operations/CreatingNewIncome.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package budgetApplication.operations;
+
+import budgetApplication.menu.userInput.*;
```
Comment (andzejc) [inline src/main/java/budgetApplication/operations/CreatingNewIncome.java:3]: dont use wildcards

**Review by andzejc — COMMENTED** [review]
great job, update own .gitignore file. You can use google-formater(download xml or install plugin) and don't forget about code formating.

File: src/main/java/budgetApplication/BudgetApplication.java (hunk 1 above)
Comment (ZilvinasVysniauskas) [reply src/main/java/budgetApplication/BudgetApplication.java]: Done

File: src/main/java/budgetApplication/menu/MainOperationStarter.java (hunk 2 above)
Comment (ZilvinasVysniauskas) [reply src/main/java/budgetApplication/menu/MainOperationStarter.java]: DONE

File: src/main/java/budgetApplication/operations/CreatingNewIncome.java (hunk 3 above)
Comment (ZilvinasVysniauskas) [reply src/main/java/budgetApplication/operations/CreatingNewIncome.java:5]: DONE

**Pull request merged**